Themis/Views/Assessment/CorrectionSidebar/GeneralFeedbackCellView.swift

```diff
@@ -21,21 +22,40 @@ struct GeneralFeedbackCellView: View {
                         .font(.title3)
                     Spacer()
                     Button {
-                        showAddFeedback = true
+                        feedbackType = .general
+                        showAddFeedback.toggle()
                     } label: {
                         Image(systemName: "plus")
                     }
                 }.padding()) {
-                    ForEach(assessment.feedback.feedbacks) { feedback in
+                    ForEach(assessment.feedback.feedbacks.filter { $0.type == .general }) { feedback in
```

Comment:
AssessmentModel already contains a computed property that filters the general feedbacks. Use that one instead